volatility/framework/plugins/mac/check_sysctl.py

```diff
@@ -109,30 +111,28 @@ def _process_sysctl_list(self, kernel, sysctl_list, recursive = 0):
             except exceptions.InvalidAddressException:
                 break
 
-    def _generator(self):
+    def _generator(self, mods: Iterator[Any]):
```

Comment:
Again, odd to parameterize the private `_generator` method if it doesn't get called more than once.

Reply:
I moved the module gathering for all plugins into the generator with my latest commit